### ISSUE TITLE
Unimportant articles in Football selection welcome page

### ISSUE THREAD

**Issue by Popolechien** [openzim/wp1_selection_tools#15]
The latest Football selection has a couple of false positives on the landing page (ie. top 100 articles):
- Niels Bohr;
- Drew Carey;
- George Soros.

Obviously these may have some relationship to football but probably aren't great at it (or worth making it to the top 100).

**Comment by kelson42** [comment]
@Popolechien Indeed. What should we do, remove all article with low football importance from the football selection?

**Comment by kelson42** [comment]
Yes, we should somehow have a topic specific score

**Comment by Popolechien** [comment]
That would probably be the easy way out but if that rating is based on enwp then it would negatively impact other languages (e.g. a German or Spanish team might get low ranking while it would be high on its native wiki).

Can't we simply say that low-ranking articles should be excluded from the landing page/Top100?

**Comment by kelson42** [comment]
Scores are wiki specific. We produce now selections sorted by scores. The solution you propose does not really fit to the current architecture as we have no way to directly control what is on the welcome page, this is just the first 100 elements of the selection. To me this bug is just the symptom that the project article list is sorted on generic criterias whereas the project centric evaluation should be prominent.

**Comment by Popolechien** [comment]
Looks a lot (yet different) like #16 and in the end looks mostly solved by now.

**Comment by kelson42** [comment]
No, no, the real solution is not implemented.
